scholia/scrape/ojs.py

```diff
@@ -319,7 +319,23 @@ def _fields_to_content(fields):
     title = _fields_to_content(['citation_title', 'DC.Title',
                                 'DC.Title.Alternative'])
     if title is not None:
-        entry['title'] = title
+        original_title = title  # Store the original title
```

Comment:
You have two variables `original_title` and `title` that contains the same. I do not think the `original_title` variable is necessary.

Reply:
i did it for extra clarification, but i removed it anyways